### ISSUE TITLE
Cannot read properties of null (reading 'source')

### ISSUE THREAD

**Issue by Dhaulagiri** [ef4/babel-import-util#5]
Version 1.2.0 of this package has, I think?, introduced a bug we noticed in [this CI run](https://github.com/hashicorp/design-system/runs/5978299253?check_suite_focus=true#step:5:21) as an example:

```
/home/runner/work/design-system/design-system/packages/components/@hashicorp/ember-flight-icons/components/flight-icon.js: Cannot read properties of null (reading 'source')TypeError: /home/runner/work/design-system/design-system/packages/components/@hashicorp/ember-flight-icons/components/flight-icon.js: Cannot read properties of null (reading 'source')
```

Pinning to 1.1.0 seems to fix the issue.

**Comment by ef4** [comment]
#6 was enough to fix this for some cases, but it's still possible for other plugins to trigger the bug. Full fix will be in #7. I think we just can't trust this babel API given how inconsistent plugins are about how they mutate AST.

**Comment by Dhaulagiri** [comment]
Thanks for the quick fix!
